src/ucp/wireup/wireup_cm.c

```diff
@@ -993,6 +1004,15 @@ ucs_status_t ucp_ep_client_cm_connect_start(ucp_ep_h ucp_ep,
     if (status != UCS_OK) {
         return status;
     }
+    memset(&wireup_ep->cm_local_sockaddr, 0, sizeof(wireup_ep->cm_local_sockaddr));
```

Comment:
Why do we need this memset? Please remove if not really needed

Reply:
There was no such field mask or flag to check if had a valid cm_local_sockaddr so set all to 0 for check. Added comment "set cm_local_sockaddr all 0 to check if it is valid before create uct_ep".